activities/documentation/about-folder-structure.md

```diff
@@ -6,8 +6,14 @@ type: Resource
 
 About has a folder structure that serves as a basic skeleton for our information.
 
+As any organization is formed by the information structure it is up to all of us to make sure we make the documentation we write for one specific use case easy to be incorporated in other cases.
+
+In order to prevent silos we don't split our activities with the roles that most commonly do them. This helps us share more activities, create communities of practice and remove duplication wherever possible.
```

Comment:
I really like this.

Reply:
This policy has a sharp tension in it. Does it discourage writing step by step guides to solve a specific task or not? My example is the community calls, which are recurring and has tens of different steps that we want to do exactly the same each time. If we write it generic, it will increase the cognitive task of executing it every time, making it slower and more prone to error. Sure, it will make it easier to get an overview for anyone who does not need to actually use the instructions, but will not make it easier to do the actual task.

Reply:
That is not at all the intent.

My assumption here was that the `community-calls` guidance was for all community calls, not just the Standard community call.

Ideally, we have an activity on community calls, where the things that are shared live, and the implementations where they are needed.

For example:

* `activities/community-calls`: general things on community calls
* `activities/standard-maintainance/community-call.md`: precise steps for Standard for Public Code call
* `activities/communications/community-call.md`: precise steps and guidance for Foundation for Public Code community call

These can all be linked.

So you think this change should be made in this pull request already.